llvm-libcxx-16.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: llvm-libcxx-16
+  version: 16.0.6
+  epoch: 0
+  description: The LLVM libc++ libraries
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - llvm-libcxx=16
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - cmake
+      - samurai
+      - git
+      - python3
+      - curl
+      - clang
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/llvm/llvm-project
+      tag: llvmorg-${{package.version}}
+      expected-commit: 7cbf1a2591520c2491aa35339f227775f4d3adf6
+
+  - runs: |
+      CC=clang CXX=clang cmake -G Ninja -S runtimes -B build \
+        -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi" \
```

Comment:
This will be ABI-incompatible with libstdc++, which means that C++ libraries shipped in Wolfi can't be used with libc++.

I think, instead, what we want is `-DLIBCXX_CXX_ABI=libstdc++`, which gives us the extended functionality in libc++, but in a way which defers to libstdc++ for common functionality.

Reply:
Got it!